### PR TITLE
Ensure New People Groups Option Also Hidden If Disabled

### PR DESCRIPTION
**Pull request by kodinkat**

### CODE REVIEW

File: dt-people-groups/people-groups-base.php [hunk 1]
```diff
@@ -73,8 +73,24 @@ public function dt_details_additional_section( $section, $post_type ) {
...
 
     public function dt_nav_filter( $navigation_array ) {
+        $is_hidden = ! get_option( Disciple_Tools_People_Groups::$option_key_settings_display_tab );
+
         if ( isset( $navigation_array['main'], $navigation_array['main'][ $this->post_type ] ) ) {
-            $navigation_array['main'][ $this->post_type ]['hidden'] = ! get_option( Disciple_Tools_People_Groups::$option_key_settings_display_tab );
+            $navigation_array['main'][ $this->post_type ]['hidden'] = $is_hidden;
+        }
+
+        if ( isset( $navigation_array['admin'], $navigation_array['admin']['add_new'] ) ) {
```
Comment (corsacca) [inline dt-people-groups/people-groups-base.php:82]: @kodinkat what is happening here?

Comment (kodinkat) [reply dt-people-groups/people-groups-base.php:82]: Loops add-new submenu list and sets the people groups post type hidden flag accordingly, before updating the main navigation_array and returning.

File: dt-people-groups/people-groups-base.php [hunk 2]
```diff
@@ -73,8 +73,24 @@ public function dt_details_additional_section( $section, $post_type ) {
...
+            $updated_submenus = [];
+            foreach ( $navigation_array['admin']['add_new']['submenu'] ?? [] as $submenu ) {
```
Comment (corsacca) [inline dt-people-groups/people-groups-base.php:84]: can we add the `&` before $submenu. so we don't need the temp array?

Comment (kodinkat) [reply dt-people-groups/people-groups-base.php:84]: @corsacca Funny, as this was my first approach, to update by reference (&); but for some reason, it didn't take and did not seem happy with the nested submenus! Not to worry, as I'll circle back and try again... No doubt, it's me missing something!! 😉

File: dt-assets/parts/nav-topbar.php [hunk 3]
```diff
@@ -203,12 +203,16 @@
                         <ul class="submenu menu vertical title-bar-right add-new-items-dropdown">
                             <?php if ( isset( $dt_nav_tabs['admin']['add_new']['submenu'] ) && ! empty( $dt_nav_tabs['admin']['add_new']['submenu'] ) ) : ?>
                                 <?php foreach ( $dt_nav_tabs['admin']['add_new']['submenu'] as $dt_nav_submenu ) : ?>
-                                    <li>
-                                        <a class="add-new-menu-item" href="<?php echo esc_url( $dt_nav_submenu['link'] ) ?>">
-                                            <img title="<?php echo esc_html( $dt_nav_submenu['label'] ); ?>" src="<?php echo esc_url( $dt_nav_submenu['icon'] ?? get_template_directory_uri()  . '/dt-assets/images/circle-add-green.svg' ) ?>">
-                                            <?php echo esc_html( $dt_nav_submenu['label'] ); ?>
-                                        </a>
-                                    </li>
+                                    <?php if ( isset( $dt_nav_submenu['hidden'] ) && ! $dt_nav_submenu['hidden'] ) { ?>
```
Comment (corsacca) [inline dt-assets/parts/nav-topbar.php:206]: @kodinkat we might run into an issue again if hidden is not set.

**Comment by corsacca** [comment]
Thanks @kodinkat !

**Pull request merged**